exercises/Version-control-basic-set-up-git-R.md

```diff
@@ -28,11 +28,25 @@ before somehow hitting `Ctrl-s` and overwriting all of your hard word.
 Determined to not let this happen again you've committed to using `git` for
 version control.
 
-Install `git` for your operating system following the
+Install `Git` for your operating system following the
 [setup instructions](http://www.datacarpentry.org/semester-biology/computer-setup/). Then
-create a new project for this assignment in RStudio with the following steps:
-
-1. File -> New Project -> New Directory -> Empty Project.
-2. Choose where to put your project.
-3. Select `Create a git repository`.
-4. If everything worked in the upper right corner of RStudio you should see a `Git` tab.
+create a new repo at the Github organization for the class:
+
+1. Navigate to Github in a web browser and login.
+2. Click the `+` at the upper right corner of the page that flashes `Create 
+   new...` and choose `New repository`.
+3. Choose the class organization (e.g., `dcsemesterYEAR`) as the `Owner` of the
```

Comment:
This should just be `dcsemester`. Otherwise I have to apply for a new course repo every year.